pkg/sriov/sriov_test.go

```diff
@@ -102,18 +102,56 @@ var _ = Describe("Sriov", func() {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(netconf.HostIFGUID).To(Equal(hostGUID))
 		})
-		It("ApplyVFConfig without GUID", func() {
+		It("ApplyVFConfig without GUID, VF's GUID all zeroes", func() {
 			mockedNetLinkManger := &mocks.NetlinkManager{}
 			mockedPciUtils := &mocks.PciUtils{}
 
-			fakeLink := &FakeLink{}
+			hostGUID := "00:00:00:00:00:00:00:00"
+			gid, err := net.ParseMAC("00:00:04:a5:fe:80:00:00:00:00:00:00:" + hostGUID)
+			Expect(err).ToNot(HaveOccurred())
+
+			fakeLink := &FakeLink{netlink.LinkAttrs{
+				HardwareAddr: gid,
+			}}
 			mockedNetLinkManger.On("LinkByName", mock.AnythingOfType("string")).Return(fakeLink, nil)
 
 			sm := sriovManager{nLink: mockedNetLinkManger, utils: mockedPciUtils}
-			err := sm.ApplyVFConfig(netconf)
+			err = sm.ApplyVFConfig(netconf)
+			Expect(err).To(HaveOccurred())
+		})
+		It("ApplyVFConfig without GUID, VF's GUID all 'F's", func() {
+			mockedNetLinkManger := &mocks.NetlinkManager{}
+			mockedPciUtils := &mocks.PciUtils{}
+
+			hostGUID := "ff:ff:ff:ff:ff:ff:ff:ff"
+			gid, err := net.ParseMAC("00:00:04:a5:fe:80:00:00:00:00:00:00:" + hostGUID)
+			Expect(err).ToNot(HaveOccurred())
+
+			fakeLink := &FakeLink{netlink.LinkAttrs{
+				HardwareAddr: gid,
+			}}
+			mockedNetLinkManger.On("LinkByName", mock.AnythingOfType("string")).Return(fakeLink, nil)
+
+			sm := sriovManager{nLink: mockedNetLinkManger, utils: mockedPciUtils}
+			err = sm.ApplyVFConfig(netconf)
+			Expect(err).To(HaveOccurred())
+		})
+		It("ApplyVFConfig without GUID, VF's GUID is configured", func() {
+			mockedNetLinkManger := &mocks.NetlinkManager{}
+			mockedPciUtils := &mocks.PciUtils{}
+
+			hostGUID := "11:22:33:00:00:aa:bb:cc"
+			gid, err := net.ParseMAC("00:00:04:a5:fe:80:00:00:00:00:00:00:" + hostGUID)
+			Expect(err).ToNot(HaveOccurred())
+
+			fakeLink := &FakeLink{netlink.LinkAttrs{
+				HardwareAddr: gid,
+			}}
+			mockedNetLinkManger.On("LinkByName", mock.AnythingOfType("string")).Return(fakeLink, nil)
+
+			sm := sriovManager{nLink: mockedNetLinkManger, utils: mockedPciUtils}
+			err = sm.ApplyVFConfig(netconf)
 			Expect(err).NotTo(HaveOccurred())
-			Expect(netconf.HostIFGUID).To(Equal(""))
```

Comment:
why were these two lines removed ? 

i think i added this comment in the previous version of this PR

Reply:
or maybe i didnt :(

anyway apart from this comment im pretty much /LGTM

Reply:
> why were these two lines removed ?

Original test case assumed that if error occurred then netconf will be modified. New code logic don't touch netconf at all, so no need to check if it's left empty.